### PR TITLE
Fix https://github.com/esphome/issues/issues/658

### PR DESCRIPTION
**Pull request by fake-name**
See https://github.com/esphome/issues/issues/658

### CODE REVIEW

**Comment by glmnet** [comment]
Should should have filled in the PR template, you briefly explain there the PR information and you can link to other issues.

File: esphome/components/web_server/web_server.cpp [hunk 1]
```diff
@@ -7,7 +7,10 @@
 #include "StreamString.h"
 
 #include <cstdlib>
-#include <esphome/components/logger/logger.h>
+
+#ifdef USE_LOGGER
+  #include <esphome/components/logger/logger.h>
```
Comment (glmnet) [inline esphome/components/web_server/web_server.cpp]: lint is failing here because the `#include` should not be indented inside the `#idfef`

**Review by glmnet — CHANGES_REQUESTED** [review]
Please fix for travis build

File: esphome/components/web_server/web_server.cpp (hunk 1 above)
Comment (fake-name) [reply esphome/components/web_server/web_server.cpp]: Ugh, that code style is so gross.

**Review by glmnet — APPROVED** [review]

**Pull request merged**